python/dataAPIutilities.py

```diff
@@ -4,48 +4,16 @@
 """
```

Comment:
Agree with your suggestion to get rid of this package and move the remaining stuff to tstUtilities.py

TAG_FOR_INIT can go there too, no need for a separate file.

Reply:
Doing this will create a dependency between `vTestInterface.py` and `tstUtilities.py` -- are you okay with that?